spec/src/main/asciidoc/core/definition.asciidoc

```diff
@@ -493,13 +493,14 @@ Alternatively, a scope type may be specified using a stereotype annotation, as d
 
 ==== Default scope
 
-When no scope is explicitly declared by annotating the bean class or producer method or field the scope of a bean is defaulted.
+When no scope is explicitly declared by annotating the bean class or producer method or field and the bean is not deployed in a <<trimmed_bean_archive>> the scope of a bean is defaulted.
```

Comment:
I think what's meant here is that if you `trim` the archive, the classes without annotations will not actually become bean classes (you will veto them) and therefore you don't need to explicitly state that the scope will not be defaulted. Hence this change seems unnecessary.